network-api/networkapi/mozfest/templates/mozfest/404.html

```diff
@@ -3,7 +3,8 @@
 {% load i18n %}
 
 {% block page_title %}
-{% blocktrans %}Mozilla Festival - Page Not Found{% endblocktrans %}
+  {% if APP_ENVIRONMENT == "Staging" %} [S] {% elif APP_ENVIRONMENT == "Review" %} [RA] {% endif %}
```

Comment:
Since we do the same thing in six different places, let's make this either its own includable fragment, or a template tag like `{% environment_prefix %}`

Reply:
Done!